src/components/PlanRouteButton/index.jsx

```diff
@@ -0,0 +1,20 @@
+function PlanRouteButton({boolean, setBoolean}) {
+
+    const clickHandler = () => {
+        if (boolean === true) {
```

Comment:
Again, better to rename this to `buttonClicked` or `buttonClickedToggle` to make it clear what this does